### PR TITLE
add ternoa network to the list of testnets

### PR DESCRIPTION
**Pull request by ETeissonniere**
Ternoa is a NFT focused network with unique features such as confidential data linked to NFTs and an upcoming marketplace.
The Chaos Network serves as Ternoa's test network, would be great to have it accessible from Polka JS instead of our own fork.

### CODE REVIEW

File: packages/apps-config/src/endpoints/testing.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ import { expandEndpoints } from './util';
 //   text: The text to display on the dropdown
 //   value: The actual hosted secure websocket endpoint
 
-export function createTesting (t: TFunction): LinkOption[] {
+export function createTesting(t: TFunction): LinkOption[] {
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/testing.ts:18]:
```suggestion
export function createTesting (t: TFunction): LinkOption[] {
```

File: packages/apps-config/src/endpoints/testing.ts [hunk 2]
```diff
@@ -197,6 +197,14 @@ export function createTesting (t: TFunction): LinkOption[] {
         Soramitsu: 'wss://ws.stage.sora2.soramitsu.co.jp'
       }
     },
+    {
+      dnslink: 'ternoa-chaos',
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/testing.ts:201]:
```suggestion
```

**Review by jacogr — COMMENTED** [review]
Thank you! Always happy to add chains.

Some small comments.

**Comment by ETeissonniere** [comment]
Sounds good! Just applied your requested changes

**Review by jacogr — APPROVED** [review]
Thank you!

**Review by jacogr — APPROVED** [review]
Thank you!

**Pull request merged**